pkg/planner/core/logical_plans_test.go

```diff
@@ -2270,7 +2377,124 @@ func testColumnPrivilegeVisitInfo(t *testing.T) {
 		require.NoError(t, err, tt.sql)
 		checkVisitInfo(t, builder.visitInfo, tt.ans, tt.sql)
 
-		require.Nil(t, TryFastPlan(s.ctx, nodeW))
+		require.Nil(t, TryFastPlan(s.ctx, nodeW), tt.sql)
```

Comment:
All cases in `testColumnPrivilegeVisitInfo` are not (Batch)PointGet, we check them here